subscription_management/lambda_function.py

```diff
@@ -255,7 +255,7 @@ def lambda_handler(event, context):
             from twilio.rest import Client
             client = Client(TWILIO_API_SID, os.getenv('TWILIO_API_KEY'))
             confirm_msg = "Bug Alert: you are opted in. SMS & calls will come from this number, be sure to save it to your contacts."
-            client.messages.create(body=confirm_msg, from_=TWILIO_MSG_SERVICE, to='+' + str(phone_country_code) + str(phone_number))
+            client.messages.create(body=confirm_msg, from_=TWILIO_MSG_SERVICE, to='+' + str(phone_country_code) + str(phone_number), max_price=0.11)
```

Comment:
Will this throw an exception if `max_price` is exceeded? If so, it could be valuable to log (or not log) the failure reason.

> All messages are queued for delivery and the message cost is checked before the message is sent. If the cost exceeds max_price, the message will fail and a status of Failed is sent to the status callback.

https://www.twilio.com/docs/sms/api/message-resource#create-a-message-resource
https://www.twilio.com/docs/libraries/python#handling-exceptions

Reply:
Yeah, as-written it will raise an unhandled exception and 500 the API Gateway server response. I'll wrap in try/except.